### PR TITLE
Add coverage for Bz 2210289

### PR DESCRIPTION
**Pull request by vkathole**
https://bugzilla.redhat.com/show_bug.cgi?id=2210289

### CODE REVIEW

**Review by mashetty330 — COMMENTED** [review]
LGTM, please run a validation job

File: tests/manage/mcg/test_bucket_policy.py [hunk 1]
```diff
@@ -979,3 +979,63 @@ def test_multipart_with_policy(self, mcg_obj, bucket_factory):
         # Completing the Multipart Upload
         logger.info(f"Completing the MP Upload with on bucket: {bucket}")
         complete_multipart_upload(obc_obj, bucket, object_key, upload_id, uploaded_part)
+
```
Comment (PrasadDesala) [inline tests/manage/mcg/test_bucket_policy.py:982]: polarion id missing

File: tests/manage/mcg/test_bucket_policy.py [hunk 2]
```diff
@@ -979,3 +979,63 @@ def test_multipart_with_policy(self, mcg_obj, bucket_factory):
...
+        obc_name = bucket_factory(amount=1, interface="OC")[0].name
+        obc_obj = OBC(obc_name)
+
+        actions_list = [
+            "GetBucketObjectLockConfiguration",
+            "GetObjectRetention",
+            "GetObjectLegalHold",
+            "PutBucketObjectLockConfiguration",
+            "PutObjectRetention",
+            "GetObjectLegalHold",
```
Comment (PrasadDesala) [inline tests/manage/mcg/test_bucket_policy.py:999]: ListObjects 
ListMultipartUploads
GetObjectAttributes
ListObjectVersions
ListParts
PutObjectRetention
UploadPart
UploadPartCopy

are all the above operations covered here?

File: tests/manage/mcg/test_bucket_policy.py [hunk 3]
```diff
@@ -979,3 +979,63 @@ def test_multipart_with_policy(self, mcg_obj, bucket_factory):
...
+        put_policy = put_bucket_policy(mcg_obj, obc_obj.bucket_name, bucket_policy)
+
+        if put_policy is not None:
+            response = HttpResponseParser(put_policy)
+            if response.status_code == 200:
+                logger.info("Bucket policy has been created successfully")
+            else:
+                raise InvalidStatusCode(f"Invalid Status code: {response.status_code}")
+        else:
+            raise NoBucketPolicyResponse("Put policy response is none")
```
Comment (PrasadDesala) [inline tests/manage/mcg/test_bucket_policy.py:1019]: you could actually throw an assertion here

**Review by PrasadDesala — APPROVED** [review]

**Review by mashetty330 — APPROVED** [review]

**Pull request merged**